<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunSlurmDialog</class>
 <widget class="QDialog" name="RunSlurmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slurm Run Parameters</string>
  </property>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Run with Slurm</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>110</y>
     <width>94</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentText">
    <string>00:30:00</string>
   </property>
   <item>
    <property name="text">
     <string>00:30:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>01:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>02:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>05:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>08:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>36:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>48:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>54:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>72:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>84:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>96:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>120:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>144:00:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>168:00:00</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox_nodes">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>40</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>43</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of nodes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ppn">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>74</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>110</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Walltime (HH:mm:ss)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Processors per node</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
